app/init.php

```diff
@@ -344,6 +361,13 @@ function($value) {
 ]);
 
 // Runtime Execution
+App::setResource('logger', function($register) {
+    return $register->get('logger');
```

Comment:
I think this can throw a runtime exception if logger has not been set. Can you confirm this ? We might need to wrap it in a try/ catch in that case

Reply:
From what I can see, it would only throw an error if `$register->set('logger',` did not run, but we run that all the time so it should be good.

It can return `null` tho, so we need to make sure the logger is set. We do that in all API, real-time and worker - on all 3 places.

The only moment we throw an error is when the user defines a provider that is not supported, but that makes sense.

Reply:
Just tested it to double-check and all seems to be working fine.